etc/eslint/rules/typescript.js

```diff
@@ -0,0 +1,2452 @@
+*/
+rules[ 'one-var' ] = [ 'error', 'never' ];
+
+/**
+* Disables the rule for padding lines between statements.
+*
+* @name padding-line-between-statements
+* @memberof rules
+* @type {Array}
+* @default [ 'off', { 'blankLine': 'always', 'prev': '*', 'next': 'return' } ]
+* @see [padding-line-between-statements]{@link https://eslint.org/docs/rules/padding-line-between-statements}
+*/
+rules[ 'padding-line-between-statements' ] = [ 'off', {
+	'blankLine': 'always',
+	'prev': '*',
+	'next': 'return'
+}];
+
+/**
+* Requires the use of `const` for variables that are never reassigned after declared.
+*
+* @name prefer-const
+* @memberof rules
+* @type {string}
+* @default 'error'
+* @see [prefer-const]{@link https://eslint.org/docs/rules/prefer-const}
+*/
+rules[ 'prefer-const' ] = 'error';
+
+/**
+* Disables the preference for using object spread over `Object.assign`.
+*
+* @name prefer-object-spread
+* @memberof rules
+* @type {string}
+* @default 'off'
+* @see [prefer-object-spread]{@link https://eslint.org/docs/rules/prefer-object-spread}
+*/
+rules[ 'prefer-object-spread' ] = 'off';
+
+/**
+* Disables the preference for template literals over string concatenation.
+*
+* @name prefer-template
+* @memberof rules
+* @type {string}
+* @default 'off'
+* @see [prefer-template]{@link https://eslint.org/docs/rules/prefer-template}
+*/
+rules[ 'prefer-template' ] = 'off';
+
+/**
+* Enforces quote properties in object literals to be always present.
+*
+* @name quote-props
+* @memberof rules
+* @type {Array}
+* @default [ 'error', 'always' ]
+* @see [quote-props]{@link https://eslint.org/docs/rules/quote-props}
+*/
+rules[ 'quote-props' ] = [ 'error', 'always' ];
+
+/**
+* Disables the enforcement of the consistent use of either backticks, double, or single quotes.
+*
+* @name quotes
+* @memberof rules
+* @type {string}
+* @default 'off'
+* @see [quotes]{@link https://eslint.org/docs/rules/quotes}
+*/
+rules[ 'quotes' ] = 'off';
+
+/**
+* Enforces passing the radix parameter to the `parseInt()` function.
+*
+* @name radix
+* @memberof rules
+* @type {string}
+* @default 'error'
+* @see [radix]{@link https://eslint.org/docs/rules/radix}
+*
+* @example
+* // Bad...
+* const bad = parseInt( '123' );
+*
+* // Good...
+* const good = parseInt( '123', 10 );
+*/
+rules[ 'radix' ] = 'error';
+
+/**
+* Disables the requirement for `await` in `async` functions.
+*
+* @name require-await
+* @memberof rules
+* @type {string}
+* @default 'off'
+* @see [require-await]{@link https://eslint.org/docs/rules/require-await}
+*/
+rules[ 'require-await' ] = 'off';
+
+/**
+* Enforces semicolons at the end of each statement instead of reliance on ASI.
+*
+* @name semi
+* @memberof rules
+* @type {string}
+* @default [ 'error', 'always' ]
+* @see [semi]{@link https://eslint.org/docs/rules/semi}
+*
+* @example
+* // Bad...
+* const bad = 'no semicolon'
+*
+* // Good...
+* const good = 'semicolon';
+*/
+rules[ 'semi' ] = [ 'error', 'always' ];
```

Comment:
@Planeshifter Isn't this a duplicate of https://github.com/stdlib-js/stdlib/pull/1126/files#diff-e34afd8893f1cb8f746b105084f21b243edcdab58bdbea94c70bbaaa7ed583c6R942?

Reply:
Aw shoot, this was why I had it disabled, because the semicolons would already be enforced by the TS equivalent. Will remove this rule setting altogether as it is confusing.